docs/references/go/overview.mdx

```diff
@@ -5,32 +5,112 @@ description: Learn how to integrate Go into your Clerk application.
 
 # Go
 
-Clerk's Go SDK is a thin wrapper over the Clerk Backend API. Add the following import statement:
+Clerk's [Go SDK](https://github.com/clerk/clerk-sdk-go) is a wrapper over the [Clerk Backend API](https://clerk.com/docs/reference/backend-api).
+
+## Installation
+
+If you are using Go Modules and have a `go.mod` file in your project's root, you can import clerk-sdk-go directly.
 
 ```go
-import "github.com/clerkinc/clerk-sdk-go/clerk"
+import (
+    "github.com/clerk/clerk-sdk-go/v2"
+)
 ```
 
-Go doesn't automatically add the module. You can explicitly add it with:
+Alternatively, you can `go get` the package explicitly.
 
-```go
-$ go get github.com/clerkinc/clerk-sdk-go
+```sh filename="terminal"
+go get -u github.com/clerk/clerk-sdk-go/v2
 ```
 
-## `Clerk` client
+## Usage
+
+For details on how to use this module, see the [Go Documentation](https://pkg.go.dev/github.com/clerk/clerk-sdk-go/v2).
+
+The library has a resource-based structure which follows the way the [Clerk Backend API](https://clerk.com/docs/reference/backend-api) resources are organized.
+Each API supports specific operations, like Create or List. While operations for each resource vary, a similar pattern is applied throughout the library.
+
+In order to start using API operations the library needs to be configured with your Clerk API secret key. Depending on your use case,
+there's two ways to use the library; with or without a client.
+
+For most use cases, the API without a client is a better choice.
+
+On the other hand, if you need to set up multiple Clerk API keys, using clients for API operations provides more flexibility.
```

Comment:
Why would one need to setup multiple API keys?

Reply:
I don't have a specific use-case in mind. Perhaps I can change the sentence to "if you need more control and flexibility over the library's configuration"?

Would that make more sense?

Reply:
Maybe change the sentence to explain that you can instanciate multiple clients in order to handle use cases like wanting to change multiple sites (requiring multiple API keys)?

Reply:
Updated. Thanks for the suggestion!